README.md

```diff
@@ -45,6 +45,7 @@ The following Shelly devices are supported:
 * [Shelly Sense](https://shelly.cloud/shelly-sense/)
 * Shelly Smoke
 * [Shelly Smoke 2](https://shelly.cloud/products/shelly-smoke-smart-home-automation-sensor/)
+* [Shelly Uni](https://shelly.cloud/products/shelly-uni-smart-home-automation-device/)
```

Comment:
Please also add the Shelly 1L to the readme.

Reply:
Done.